lib/fog/aws/storage.rb

```diff
@@ -284,15 +314,15 @@ def request_params(params)
               path_style = params.fetch(:path_style, @path_style)
               if !path_style
                 if COMPLIANT_BUCKET_NAMES !~ bucket_name
-                  Fog::Logger.warning("fog: the specified s3 bucket name(#{bucket_name}) is not a valid dns name, which will negatively impact performance.  For details see: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html")
+                  Fog::Logger.warning("fog: the specified s3 bucket description(#{bucket_name}) is not a valid dns description, which will negatively impact performance.  For details see: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html")
```

Comment:
The error changes here from using "name" to "description" seem unrelated to other changes, and I worry they might be somewhat confusing/surprising. Could you help me understand them and/or drop them for now to keep this focused? Thanks!

Reply:
Whoops, I'm notsure how this line got changed. Will revert it.

Reply:
Fixed